tensorboard/webapp/metrics/effects/index.ts

```diff
@@ -76,6 +106,22 @@ export class MetricsEffects implements OnInitEffects {
     private readonly dataSource: MetricsDataSource
   ) {}
 
+  /**
+   * Computes a record of tag to the experiments it appears in.
+   *
+   * The computation is done by translating Plugin -> Tag -> Run -> ExpId
+   *
+   * Sampled plugins are ignored because they are associated with runs, not experiments.
```

Comment:
Sampled plugins are not the only things ignored here. Really it's any single-run plugins that are ignored. Would be good to fix that in the documentation.

I think its also worth explaining the real motivation for the change here - otherwise it is hard to understand why we would bother doing this for scalars and why we wouldn't do this for the others: 

We want to eliminate unnecessary requests for experiment+tag combinations where the experiment does not actually contain the tag.  In case of single-run plugins we assume that every given request for expeirment+run+tag is already valid, since they originate from cards for that experiment+run+tag combination.